src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,603 @@
+﻿// <copyright file="AuthorizationServersScenarios.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    [Collection(nameof(AuthorizationServersScenarios))]
+    public class AuthorizationServersScenarios
+    {
+        private const string SdkPrefix = "dotnet_sdk";
+
+        [Fact(Skip = "Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
```

Comment:
We should be able to enable key tests once the new key storage feature is rolled out (OKTA-303121).

Reply:
Awesome 👍 I'll note the ticket number for visibility.